seatunnel-engine/seatunnel-engine-server/pom.xml

```diff
@@ -33,6 +33,11 @@
             <artifactId>seatunnel-engine-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I I suggest that we can move [multitablesink](https://github.com/apache/seatunnel/tree/c645d925dcb85d6d0932427eba39a7e4283ec8ed/seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/multitablesink) package to seatunnel-common module instead of referencing connector-common. Because our connector is engine independent

Reply:
move multitablesink package to seatunnel-common module, will the impact be a little big

Reply:
Never mind. we have enough test case.

Reply:
The classes in multitablesink package refer to the seateunel-api module ,  which when moved causes seatunnel-common and seatunnel-api loop dependencies.